.github/workflows/bindings_java.yml

```diff
@@ -81,3 +81,14 @@ jobs:
         run: |
           ./mvnw clean install -DskipTests
           ./mvnw verify artifact:compare
+      - name: Behavior tests
+        working-directory: bindings/java
+        run: |
+          export OPENDAL_TEST=memory
+          export OPENDAL_FS_ROOT=/opendal
```

Comment:
Maybe this should be `OPENDAL_MEMORY_ROOT`?

Reply:
Interesting. Why test not fail?

Reply:
> Interesting. Why test not fail?

root for every service default to `/`.

Reply:
Updated.